### ISSUE TITLE
Can't use the gradle plugin in an Android project

### ISSUE THREAD

**Issue by villevoutilainen** [casid/jte#384]
The instructions suggest requiring the java plugin as a dependency. The plugin seems to do that itself, too.
A gradle project can't use both the java plugin and the android plugin.

I do not need to *run* JTE on Android. I need to run it during my Android project's build. I want to precompile my templates,
and use them in an annotation processor that also runs during the build, not on the target phone.

Any ideas?

**Comment by kelunik** [comment]
What are you trying to do with jte then? Generate code within the build process?

**Comment by villevoutilainen** [comment]
Yeah. The build process runs an annotation processor, which then uses JTE to generate code from those annotations. To be more specific, this is https://github.com/villevoutilainen/Jenny

**Comment by edward3h** [comment]
I don't know about Android builds.

I'm imagining your annotation processor depends on JTE plugin at build time. Then the main code depends on the annotation processor at its build time. So the main code build doesn't depend on the JTE plugin.

Does your annotation processor need to be built with the Android toolchain?
